src/non_uniform_conformal_cubed_sphere.jl

```diff
@@ -0,0 +1,409 @@
+using LinearAlgebra
+using Statistics: mean, norm
+using Random
+
+
+function compute_deviation_from_isotropy_at_point(X, Y, i, j)
+    x1 = X[i, j]
+    y1 = Y[i, j]
+    x2 = X[i+1, j]
+    y2 = Y[i+1, j]
+    x3 = X[i+1, j+1]
+    y3 = Y[i+1, j+1]
+    x4 = X[i, j+1]
+    y4 = Y[i, j+1]
+
+    # Compute the vectors.
+    v1 = [x2 - x1, y2 - y1]
+    v2 = [x3 - x2, y3 - y2]
+    v3 = [x4 - x3, y4 - y3]
+    v4 = [x1 - x4, y1 - y4]
+
+    # Compute the deviation from isotropy.
+    deviation_from_isotropy_at_point = (
+    abs(norm(v1) - norm(v2)) + abs(norm(v2) - norm(v3)) + abs(norm(v3) - norm(v4)) + abs(norm(v4) - norm(v1)))
+
+    return deviation_from_isotropy_at_point
+end
+
+
+function compute_deviation_from_isotropy(X, Y)
+    Nx, Ny = size(X)
+    deviation_from_isotropy = zeros(Nx-1, Ny-1)
+
+    for i in 1:Nx-1
+        for j in 1:Ny-1
+            deviation_from_isotropy[i, j] = compute_deviation_from_isotropy_at_point(X, Y, i, j)
+        end
+    end
+
+    return norm(deviation_from_isotropy)
+end
+
+
+function compute_cell_area(X, Y, i, j)
```

Comment:
is this function different from either of 

https://github.com/CliMA/Oceananigans.jl/blob/30e3e05fe661cb5f7a0db37ab6c3f42ac8c5f80e/src/Grids/grid_utils.jl#L321-L375

?

Reply:
@navidcy, thanks for pointing me to those functions---they’re indeed the better choice as they account for the spherical surface in computing quadrilateral areas. I've updated the implementation to use these functions for calculating cell areas in commit [78076c7](https://github.com/CliMA/CubedSphere.jl/pull/41/commits/78076c7fe6f1e639945d52e607011ed4e8e540e5).